### DISCUSSION TITLE
Modify file title and make it an alias

### DISCUSSION THREAD

**Discussion by dandobos** [SilentVoid13/Templater#940] [Help] [answered]
Hi. I have a template that is triggered when I use (Concept) at the end, for example: Experience bias (Concept)

I want the alias to be: Experience bias

How can I insert that in such that I have <% tp.file.title %> but I delete the "(Concept)" part for the alias?

Thank you!

**Comment by Zachatoo** [comment, marked as answer]
You can use the `replace()` string method to replace `"(Concept)"` with `""` (nothing).

```js
<% tp.file.title.replace("(Concept)", "") %>
```

**Comment by dandobos** [comment]
Thanks Zach... I don't want to change the file title, though, I just want to add an alias.

**Reply by Zachatoo** [reply]
My answer will not change the notes title, it will just print out the title, but modified.

**Reply by dandobos** [reply]
Ah - amazing! It works :) Sorry, it's obvious now in hindsight :)
